rasa/core/agent.py

```diff
@@ -663,8 +663,6 @@ def train(
 
         logger.debug("Agent trainer got kwargs: {}".format(kwargs))
 
-        check_domain_sanity(self.domain)
-
```

Comment:
Can't access the line here, but no longer need to import `check_domain_sanity` in this file